connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -172,6 +173,9 @@
             + "Examples: plugin.path=/usr/local/share/java,/usr/local/share/kafka/plugins,"
             + "/opt/connectors";
 
+    public static final String CONFIG_PROVIDERS_CONFIG = "config.providers";
+    protected static final String CONFIG_PROVIDERS_DOC = "List of configuration providers.";
```

Comment:
Please be more specific. "List of configuration providers. This is a comma-separated list of the fully-qualified names of the ConfigProvider implementations, in the order they will be created, configured, and used."